src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/DiagramViewModel.java

```diff
@@ -465,4 +467,19 @@ public InputGraph next() {
             }
         };
     }
+
```

Comment:
When **deleting** a graph or **adding** a new graph to a group (e.g. when stepping through VM code in a c++ debugger) the `_isDuplicate` property should be recomputed for correctness. Luckily we have `groupContentChangedListener` that is called in exactly those two cases. I therefore suggest to add 
```java
ChangedListener<Group> groupContentChangedListener = g -> {
    this.markedDuplicates = false;
    this.hideDuplicates = false;
```
to the following line 
https://github.com/openjdk/jdk/blob/5a77e6aaadbf7e0deb31b7adae23b398c6c73562/src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/DiagramViewModel.java#L174